doc/ref/plotting_options/interactivity.ipynb

```diff
@@ -0,0 +1,297 @@
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "\n",
+    "opts = dict(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    by='species',\n",
+    "    frame_width=250,\n",
+    ")\n",
+    "\n",
+    "plot1 = df.hvplot.scatter(title='Scatter with hover tool (Default)', **opts)\n",
+    "plot2 = df.hvplot.scatter(hover=False, title='Scatter without hover tool', **opts)\n",
+    "\n",
+    "plot1 + plot2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "30839bfb-e814-42d8-bac6-d61b5b84cead",
+   "metadata": {},
+   "source": [
+    "(options-hover_cols)=\n",
+    "## `hover_cols`\n",
+    "\n",
+    "Specifies additional columns from the dataset to be shown in the hover tooltip.\n",
+    "- Accepts a list of column names, a single column name as a string, or 'all' to include all available columns.\n",
+    "- When set to 'all', it includes index columns only if `use_index=True`.\n",
+    "- Ignored for `datashade=True` plots, as those do not support interactivity.\n",
+    "\n",
+    "::: {note} \n",
+    "`hover_cols` complements the default dimensions shown in the tooltip but does not override them.\n",
+    ":::"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "028678e4-0bbf-4eee-b39e-7aef71f1de2f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "\n",
+    "opts = dict(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    by='species',\n",
+    "    frame_width=250,\n",
+    ")\n",
+    "\n",
+    "plot1 = df.hvplot.scatter(title='Default hover_cols', **opts)\n",
+    "plot2 = df.hvplot.scatter(\n",
+    "    hover_cols=['sex', 'body_mass_g'], title='Additional hover columns', **opts\n",
+    ")\n",
+    "plot3 = df.hvplot.scatter(\n",
+    "    hover_cols='all', title='All hover columns', **opts\n",
+    ")\n",
+    "\n",
+    "plot4 = df.hvplot.scatter(\n",
+    "    hover_cols='all',\n",
+    "    use_index=False,\n",
+    "    title='All hover columns without index',\n",
+    "    **opts\n",
+    ")\n",
+    "\n",
+    "(plot1 + plot2 + plot3 + plot4).cols(2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2560fd13-ccc7-4be1-805f-08f60912c487",
+   "metadata": {},
+   "source": [
+    "Hover on the points in each of the examples to see the displayed hover information."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e2e2707-5c15-4ae5-bc4d-dc3f14bee454",
+   "metadata": {},
+   "source": [
+    "(options-hover_formatters)=\n",
+    "## `hover_formatters`\n",
+    "\n",
+    "A dictionary that controls how values appear in hover tooltips. Keys are column names or dimensions. Values must be one of:\n",
+    "- 'numeral' – for abbreviated numbers (e.g. 1k, 3.2M)\n",
+    "- 'datetime' – for date/time values\n",
+    "- 'printf' – for precision control like '%.2f'\n",
```

Comment:
Is there a way to tell a bit more about what `printf` is? I remember I digged a bit into that but that had to do with axis formatters so I'm not entirely sure it's the same thing. https://github.com/holoviz/hvplot/issues/1215#issuecomment-1854025766

Reply:
The bokeh reference guide has a good explanation I think https://docs.bokeh.org/en/latest/docs/reference/models/formatters.html#bokeh.models.PrintfTickFormatter

Reply:
It is a bit confusing this is a tick formatter (`PrintfTickFormatter`). Ah but I see in https://docs.bokeh.org/en/latest/docs/user_guide/interaction/tools.html#formatting-tooltip-fields that they also reference that page, so I guess they re-use this formatter for the tooltip.